app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,854 @@
+	<string name="low_score">Najgorszy wynik</string>
+	<string name="average_score">Średni wynik</string>
+	<string name="average_winning_score">Średnik zwycięski wynik</string>
+	<string name="average_rating">Średnia ocena</string>
+	<string name="average_rating_abbr">Śr.</string>
+	<string name="new_label">Nowy</string>
+	<string name="win">Zwycięzca</string>
+	<string name="yes">Tak</string>
+	<string name="no">Nie</string>
+	<string name="never">Nigdy</string>
+	<string name="keep">Zatrzymaj</string>
+	<string name="edit">Edytuj</string>
+	<string name="modify">Modyfikuj</string>
+	<string name="more">Więcej</string>
+	<string name="more_suffix">%,d więcej</string>
+	<string name="image">Obraz</string>
+	<string name="avatar">Avatar</string>
+	<string name="thumbnail">Miniatura</string>
+	<string name="in">w</string>
+	<string name="before">przed</string>
+	<string name="after">po</string>
+	<string name="status_of_prefix">Status %s</string>
+	<string name="title_clear_colors">Wyczyścić kolory?</string>
+	<string name="msg_clear_colors">Chcesz wyczyścić bieżące kolory przed przypisaniem nowych?</string>
+	<string name="msg_color_error_no_players">Nie ma graczy, którzy potrzebują koloru.</string>
+	<string name="msg_color_error_duplicate_player">Nie można przypisac koloru jeśli gracz jest wypisany dwukrotnie.</string>
+	<string name="msg_color_error_missing_player_name">Nie można przypisać kolorów jeśli gracz nie ma nazwy użytkownika ani innej unikatowej nazwy.</string>
+	<string name="msg_color_error_something_changed">Coś poszło okropnie, okropnie źle.</string>
+	<string name="msg_color_error_too_few_colors">Nie ma wystarczająco dużo kolorów.</string>
+	<string name="msg_color_success">Ty masz kolor i ty masz kolor. Każdy ma kolor!</string>
+	<string name="msg_fragment_error">Nie powinieneś zobaczyć tego komunikatu. Jeśli go widzisz wyślij proszę zrzut ekranu na bgg4android@gmail.com. Dzięki!</string>
+	<string name="are_you_sure_title">Na pewno?</string>
+	<string name="discard_new_message">"Na pewno chcesz odrzucić %s?"</string>
+	<string name="discard_changes_message">"Na pewno chcesz odrzucić zmiany w %s?"</string>
+	<string name="are_you_sure_delete_play">Na pewno chcesz usunąć tę rozgrywkę?</string>
+	<string name="are_you_sure_player_sort_custom_off">Chcesz włączyć automatyczne sortowanie graczy? Obecne pozycje startowe zostaną zmienione.</string>
+	<string name="are_you_sure_delete_collection_view">Na pewno chcesz usunąć ten widok kolekcji?</string>
+	<string name="are_you_sure_timer_reset">Chcesz aby zegar kontynuował tam gdzie przerwałeś czy chcesz zresetować i stracić obecny czas trwania gry?</string>
+	<string name="are_you_sure_clear_colors">Na pewno chcessz wyczyścić listę kolorów?</string>
+	<string name="are_you_sure_delete_collection_item">Na pewno chcesz usunąć tę pozycję z Twojej kolekcji?</string>
+	<string name="app_name">BoardGameGeek</string>
+	<string name="provider_label">Collection, Plays, and GeekBuddies</string>
```

Comment:
I don't know the context. I see it's used only in `AndroidManifest.xml` and can't figure out where it's shown and if I can just translate it strictly into Polish without caring about the context.

Reply:
```suggestion
	<string name="provider_label">Kolekcja, Rozgrywki i Znajomi</string>
```